### PR TITLE
Feature/get users query

### PR DESCRIPTION
**Pull request by aliroemhildt**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  
What (if any) features are you implementing?
- updated get all users query, this is now working
- updated AdminPage and MusicianCard to accept data 
- Made a few changes in SubmissionForm, Ryan will add the rest of the changes to his local branch

What (if anything) did you refactor?
- n/a

Were there any issues that arose?
- mutation is not working yet, still need to work with BE team and figure out how that should look

Is there anything that you need from your teammate?
- n/a

Any other comments, questions, or concerns?
- n/a

### CODE REVIEW

File: src/components/SubmissionForm/SubmissionForm.js [hunk 1]
```diff
@@ -25,14 +27,41 @@ const SubmissionForm = () => {
     setSongTitle(event.target.value)
   }
   const handleVideo = event => {
-    setVideo(event.target.value)
+    setVideo(URL.createObjectURL(event.target.files[0]))
```
Comment (mr-ryan12) [inline src/components/SubmissionForm/SubmissionForm.js:30]: Still researching the best way to send the video/image files. Need to determine if it's raw or convert into a blob.

File: src/queries.js [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
+const GET_ALL_USERS = gql`
+  {
+    fetchUsers {
+      id
+      name
+      profile
+      songTitle
+    }
+  }
+`;
```
Comment (mr-ryan12) [inline src/queries.js:15]: Really glad we got this working! Thank you!

File: src/queries.js [hunk 3]
```diff
@@ -1,22 +1,32 @@
...
     $genre: String!
     $songTitle: String!
     $profile: Sting!
     $video: String!
   ) {
-    addNewUser(
+    createUser(
       name: $name,
       email: $email,
       genre: $genre,
```
Comment (mr-ryan12) [inline src/queries.js:32]: Looks like the backend will let us know if/when we need to change the syntax for our mutation.

**Review by mr-ryan12 — APPROVED** [review]
Thank you Ali! Nice we hit a milestone with successfully fetching data.

**Review by oacortez — APPROVED** [review]
Thank you for creating this branch Ali! I see we're finally able to fetch some data from our backend, which is huge! I also see that we're not fully able to post and get that connected, that is something will work on throughout the upcoming days. Everything looks good to me!

**Pull request merged**